src/PageView/NearMe/NearMeGyms/NearMeGyms.js

```diff
@@ -1,42 +1,49 @@
-import React, { useState, useEffect } from 'react';
-import {
-  GoogleMap,
-  LoadScript,
-  Marker,
-} from '@react-google-maps/api';
-import './NearMeGyms.css';
-import InfoWindows from './InfoWindows/InfoWindows'
+import React, { useState, useEffect } from "react";
```

Comment:
This component is huge! Please consider moving some bits out of it by e.g. creating some children components

Reply:
Thanks for the feedback, Ricardo. Making those changes now